test/database_service/test_db_analysis_result.py

```diff
@@ -0,0 +1,164 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=missing-docstring
+
+"""Test AnalysisResult."""
+
+from unittest import mock
+import json
+
+import numpy as np
+
+from qiskit.test import QiskitTestCase
+from qiskit_experiments.database_service import DbAnalysisResultV1 as DbAnalysisResult
+from qiskit_experiments.database_service.device_component import Qubit, Resonator, to_component
+from qiskit_experiments.database_service.database_service import DatabaseServiceV1
+from qiskit_experiments.database_service.exceptions import DbExperimentDataError
+
+
+class TestDbAnalysisResult(QiskitTestCase):
```

Comment:
If these are all just unit tests you can ignore, but if at some point there are integration tests that are using the real service then it'd be good to tag experiments with something that lets us know they are from Qiskit CI testing for cleanup on the server side in case test case cleanup fails for some reason.

Reply:
These are all just unit tests. `qiskit-experiments` does not have a dependency on `qiskit-ibmq-provider`. There will be new integration tests in `qiskit-ibmq-provider`.